bluebrain/repo-patches/packages/py-imgaug/package.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyImgaug(PythonPackage):
```

Comment:
Why is this not-a-patch in `repo-patches`?

Reply:
Because it removes dependencies. The derivates of official packages can only be additive / override.